### DISCUSSION TITLE
[Feature Request] skip first cycle option

### DISCUSSION THREAD

**Discussion by rnp509** [HarbourMasters/2ship2harkinian#502] [Ideas]
Basically make it so that you don't have to do the very first cycle of the game in order to obtain the ocarina and Song of Time. You would start with those items similarly to MM Randomizer. If you want, you could take it a step further and include Deku Mask and Song of Healing, thereby skipping the Happy Mask Salesman scene as well.

**Comment by Gcenx** [comment]
Skipping the first cycle would be awesome, I'd thinking being given the Deku Mask and Song of Healing also be a nice addition.
